packages/firmware-release-config/package.json

```diff
@@ -5,15 +5,24 @@
     "license": "See LICENSE.md in repo root",
     "sideEffects": false,
     "main": "src/index",
+    "publishConfig": {
+        "main": "lib/index.js"
+    },
+    "files": [
+        "lib/",
+        "!**/*.map"
+    ],
     "scripts": {
-        "build:lib": "yarn validate-release-config && yarn sign-release-config",
+        "build:lib": "yarn sign-release-config && yarn g:rimraf -rf lib && yarn g:tsc --build tsconfig.lib.json && ../../scripts/replace-imports.sh ./lib",
         "depcheck": "yarn g:depcheck",
         "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
         "test:unit": "yarn g:jest",
         "type-check": "yarn g:tsc --build",
-        "generate-release-config": "yarn tsx ./scripts/generate-release-config.ts",
+        "generate-release-config": "yarn validate-release-confi yarn tsx ./scripts/generate-release-config.ts",
```

Comment:
typo? `validate-release-confi` and missing &

Reply:
Fixed https://github.com/trezor/trezor-suite/pull/18821/commits/681adbcf2fb191d1d66058d6dccf2623cea527ba